app/v2/models/response/amendHistoricFhlUkPiePeriodSummary/AmendHistoricFhlUkPiePeriodSummaryResponse.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.response.amendHistoricFhlUkPiePeriodSummary
+
+import play.api.libs.json.{ Json, OFormat }
+import v2.hateoas.HateoasLinks
+
+case class AmendHistoricFhlUkPiePeriodSummaryResponse(transactionReference: String)
+
+object AmendHistoricFhlUkPiePeriodSummaryResponse extends HateoasLinks {
+  implicit val format: OFormat[AmendHistoricFhlUkPiePeriodSummaryResponse] = Json.format[AmendHistoricFhlUkPiePeriodSummaryResponse]
```

Comment:
We're not sending anything back to the vendor so not sure we need this class. Or if we do (in the connector maybe) we should ignore the value in the service. So at most we need only a `Reads`, but I suspect its presence (certainly with this name) might lead others working on subsequent tasks to pass it all the way back to the controller.

Reply:
Makes sense but it will be needed (if only for the HATEOAS links) so will leave this as a placeholder

Reply:
`xxDownstreamResponse` ?